llvm/lib/MCA/InstrBuilder.cpp

```diff
@@ -33,7 +33,7 @@ InstrBuilder::InstrBuilder(const llvm::MCSubtargetInfo &sti,
                            const llvm::MCInstrAnalysis *mcia,
                            const mca::InstrumentManager &im)
     : STI(sti), MCII(mcii), MRI(mri), MCIA(mcia), IM(im), FirstCallInst(true),
-      FirstReturnInst(true) {
+      FirstReturnInst(true), CallLatency(100U) {
```

Comment:
Should we pass `CallLatency` when we construct the InstrBuilder and remove the call to `setCallLatency`?

Reply:
Thanks for pointing that out. I tried to address it.